notebooks/tfx_pipelines/walkthrough/solutions/tfx_walkthrough_vertex.ipynb

```diff
@@ -0,0 +1,1235 @@
+{
```

Comment:
<p>I would recommend removing this line referring to TFX 0.25. I think there are additional arguments that can be added now as well but not important to the lab to introduce.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>

Reply:
<p>Good catch! Done!</p>